Blockzilla/SettingsViewController.swift

```diff
@@ -378,6 +378,8 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         if whatsNew.shouldShowWhatsNew() {
             highlightsButton?.tintColor = UIConstants.colors.whatsNew
         }
+        
+        highlightsButton?.isEnabled  = whatsNew.shouldEnableWhatsNewButton
```

Comment:
I kind of like this because it means we have a single place (that `whatsNew.shouldEnableWhatsNewButton`) where we can check if there is anything to show.